Makefile.include

```diff
@@ -229,7 +229,7 @@ export PREFIX ?= $(if $(TARGET_ARCH),$(TARGET_ARCH)-)
 # Add standard include directories
 INCLUDES += -I$(RIOTBASE)/core/include -I$(RIOTBASE)/drivers/include -I$(RIOTBASE)/sys/include
 INCLUDES += -I$(RIOTCPU)/$(CPU)/include
-INCLUDES += -I$(RIOTBOARD)/$(BOARD)/include
+INCLUDES += $(patsubst %,-I%,$(wildcard $(RIOTBOARD)/$(BOARD)/include))
```

Comment:
use `$(addprefix -I,` instead of `$(patsubst %,-I%`

Reply:
Way better indeed.

Reply:
I updated and already squashed the fix as it easy to verify.